dynamodb-stream-reader/src/test/java/no/unit/nva/dynamodb/ResourceUtils.java

```diff
@@ -0,0 +1,31 @@
+package no.unit.nva.dynamodb;
+
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import nva.commons.utils.JsonUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class ResourceUtils {
+
+    /**
+     * Creates a DynamoDB event from a file.
+     *
+     * @param filename file to read json source from
+     * @return event created from file
+     */
+    public static DynamodbEvent loadEventFromResourceFile(String filename) {
+        ClassLoader classloader = Thread.currentThread().getContextClassLoader();
```

Comment:
We have implemented this in nva.commons IoUtils

Reply:
Using IoUtils now...